spec/requests/engine_spec.rb

```diff
@@ -4,20 +4,22 @@
 
 RSpec.describe "Waste Exemptions Engine", type: :request do
   describe "/start/new" do
+    let(:request_path) { "/start/new" }
```

Comment:
Can we use a path helper here?

Reply:
We could, but this also makes sure the routes are what we expect them to be. Probably a matter of opinion though! Our request specs generally specify the URL instead of using path helpers so I'd rather keep it this way for consistency 👍